crates/store/re_data_loader/src/lerobot.rs

```diff
@@ -20,16 +20,40 @@ use parquet::arrow::arrow_reader::ParquetRecordBatchReaderBuilder;
 use serde::de::DeserializeOwned;
 use serde::{Deserialize, Serialize};
 
-/// Check whether the provided path contains a Le Robot dataset.
+/// Check whether the provided path contains a `LeRobot` dataset.
 pub fn is_lerobot_dataset(path: impl AsRef<Path>) -> bool {
+    is_v1_lerobot_dataset(path.as_ref()) || is_v2_lerobot_dataset(path.as_ref())
+}
+
+/// Check whether the provided path contains a v2 `LeRobot` dataset.
+pub fn is_v2_lerobot_dataset(path: impl AsRef<Path>) -> bool {
+    let path = path.as_ref();
+
+    if !path.is_dir() {
+        return false;
+    }
+
+    // v2 `LeRobot` datasets store the metadata in a `meta` directory,
+    // instead of the `meta_data` directory used in v1 datasets.
+    has_sub_directories(&["meta_data", "data"], path)
```

Comment:
why is this check the same then as in `is_v1_lerobot_dataset`

Reply:
Wow i missed that 🤦  thanks